tests/e2e/operator-upgrade/psmdb/20-assert.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: kuttl.dev/v1
 kind: TestAssert
 # We use a very short timeout here because we already actively waited in this TestStep to ensure that the DB wasn't restarted so these manifest must be applied by now.
-timeout: 2
+timeout: 2000
```

Comment:
is this intentional? The comment indicates we need to use a short timeout. Please update the comment if the expectation has changed